cluster-scope/base/core/namespaces/enterprise-neurosystem-collaboration/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+    - namespace.yaml
+components:
+    - ../../../../components/project-admin-rolebindings/enterprise-neurosystem
```

Comment:
@erikerlandson in your request you requested for a `Medium` namespace quota, can you add the quota in this kustomization? ([Example](https://github.com/operate-first/apps/blob/307af64697d70e9aa85a5806692abda63a505bb6/cluster-scope/base/core/namespaces/ds-example-project/kustomization.yaml#L6))
If you are using the opf cli, you can use [the `-q` flag](https://github.com/operate-first/opfcli#create-project)

Reply:
can I just use `-q medium` ?